examples/as-median/assembly/index.ts

```diff
@@ -0,0 +1,71 @@
+
+import { parse, Arr, Integer, Value } from "assemblyscript-json/JSON";
+
+export function exec(msg: usize, out_ptr: usize): i32 {
+
+	let in_buf_raw = load<ArrayBuffer>(load<usize>(msg));
+	let in_buf = Uint8Array.wrap(in_buf_raw);
+
+	/* parse input buffer into an Array<i32> */
+	let arr = parse(in_buf) as Arr;
+	let value_array = arr.valueOf() as Array<Value>;
+	let len = value_array.length;
+	let sample = new Array<i32>(len);
+	for(let i = 0; i < len; i++) {
+		let entry = value_array[i] as Integer;
+		sample[i] = entry.valueOf() as i32;
+	}
+
+	let median = calc_median(len, sample);
+	let out = '{ median: ${median} }';
```

Comment:
This should probably be using `JSONEncoder`, like [here](https://github.com/near/assemblyscript-json#encoding-json)?